common.blocks/modal/_theme/modal_theme_islands.css

```diff
@@ -3,3 +3,52 @@
     border: 1px solid #fff;
     background: #f3f1e7;
 }
+
+@media screen and (max-width: 773px) {
+    .modal_theme_islands .modal__content {
+        box-sizing: border-box;
+        padding: 10px;
+        margin: 0;
+        position: fixed;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        top: 0;
+        overflow: hidden;
+    }
+
+    .input_theme_islands.input_size_xl {
+        font-size: 1.4em;
+        line-height: 22px;
+    }
+
+    .input_theme_islands.input_theme_islands .input__control,
+    .textarea_theme_islands.textarea_theme_islands.textarea_size_xl {
+        font-size: 12px;
+    }
+}
+
+/* Portrait */
+@media only screen
+and (min-device-width: 320px)
+and (max-device-width: 773px)
+and (orientation: portrait) {
+    .modal_theme_islands .modal__content {
+        padding: 10px;
+    }
+}
+
+/* Landscape */
+@media only screen
+and (min-device-width: 320px)
+and (max-device-width: 773px)
+and (-webkit-min-device-pixel-ratio: 2)
+and (orientation: landscape) {
+    .section__feedback-form .form__inner.form__inner {
+        columns: 2;
+        -webkit-columns: 2;
```

Comment:
most likely you don't need to specify vendor prefixes as there's `autoprefixer` in build config